src/components/search-dialog.tsx

```
@@ -0,0 +1,307 @@
+
+    const componentResults = componentRegistry
+      .filter(component =>
+        component.id.toLowerCase().includes(lowercaseQuery) ||
+        component.title.toLowerCase().includes(lowercaseQuery) ||
+        component.content.toLowerCase().includes(lowercaseQuery)
+      )
+      .map(result => ({
+        title: result.title,
+        media_type: '',
+        path: result.path,
+        excerpt: result.content.substring(0, 100) + '...',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)